web/webkit/src/main/scala/net/liftweb/http/LiftSession.scala

```diff
@@ -199,6 +199,41 @@ object LiftSession {
     }
   }
 
+  private[http] class DataAttrNode(liftSession: LiftSession) {
```

Comment:
Just verifying: these were merely promoted to first class objects, right? No other changes?

Reply:
Correct. I don't recall the exact issue now, but as they were defined they would not deserialize cleanly. Simply making them a class rather than an embedded object allows the deserialized object to re-instantiate as expected.

Reply:
That actually makes sense. It’s possible to run into similar issues with lift-json extraction. Cool.